tsSource/src/utils/edqGlobalIntuitive.ts

```diff
@@ -223,4 +232,68 @@ export function removeMultipleEDQSuggestion({edqSuggestionBox}: GlobalIntuitiveR
 	for (let i: number = 0; i < edqSuggestionBox.length; i++) {
 		edqSuggestionBox[i].parentNode.removeChild(edqSuggestionBox[i]);
 	}
+}
+/**
+ * Function attached to the window.EdqConfig.GLOBAL_INTUITIVE_AFTER_FORMAT_CHANGE configuration 
+ * that will allow to check if the user changed the address provided by clicking one option from Global Intuitive suggestion box; 
+ * if the address has been changed Pro Web will verify the address, if not Pro Web will not verify the address. 
+ * (Only if Pro Web and Global Intuitive are active).
+ * @param changed - Boolean that will let the workflow know if Pro Web Validation should validate the address or not
+**/
+export function changedAddress(changed) {
+	window.sfccConfig.addressChanged = changed;
+	if ((changed) && (document.querySelector("#form-submit"))) {
+		buttonCssSeetings({"formSubmitButton":window.sfccConfig.edqCurrentSubmitButton, "edqCurrentSubmitButton":document.querySelector("#form-submit")});
+	}
+}
+/**
+ * Function attached to the window.EdqConfig.GLOBAL_INTUITIVE_CALLBACK configuration 
+ * Will check if the address provided by clicking one option from Global Intuitive suggestion box has DPV; in case the DPV is "Y"
+ * Pro Web will not be active, if DPV isn't "Y" Pro Web will validate the address 
+ * (Only if Pro Web and Global Intuitive are active).
+ * @param changed - Boolean that will let the workflow know if Pro Web Validation should validate the address or not
+**/
+export function globalIntuitiveCallback(changed) {
+	window.sfccConfig.addressChanged = changed;
+	if (document.querySelector("#form-submit")) {
+		if (window.sfccConfig.edqDpvIndicator.value === 'Y') {
```

Comment:
I understand what's happening here, but I don't quite understand *why* this happens or has to happen this way. Ping me when you see this so we can discuss.